### PR TITLE
Increase coverage in several files

### PR DESCRIPTION
**Pull request by marcusmunch**
With the commits below, test coverage is increased to >85% for each file (~93% for all files combined).

Increased coverage can obviously be reached, as there still exist some lines of code covering more or less impossible (or highly impractical) edge-cases, and there could arguably be some use of `# nocov` statements, as most of them serve as imaginative sanity checks.

One such case is `table_exists()` getting >1 matches for a table despite no wildcard matching. This edge case can potentially be reached by having a schema "one.two" containing a table "three" and schema "one" containing a table "two.three". We already require some implementations (e.g. PostgreSQL) to have an existing schema "test" in order to unlock usability, so I'm personally hesitant about requiring a developer to set up two additional schemas in order to test an edge-case that may not even be possible (I don't know if schema names may contain `.`'s in any dialects).

Therefore, I'm keeping the >1 check in `table_exists()` for now.

### CODE REVIEW

File: tests/testthat/test-Logger.R [hunk 1]
```diff
@@ -120,3 +120,29 @@ test_that("Logger works", { for (conn in conns) { # nolint: brace_linter
     purrr::keep(~ endsWith(., ".log")) |>
     purrr::walk(~ unlink(file.path(log_path, .)))
 }})
+
+test_that("Logger stops if file exists", { for (conn in conns){
```
Comment (RasmusSkytte) [inline tests/testthat/test-Logger.R:124]: This line will likely give a linting issue.

I suggest to write instead:
test_that("Logger stops if file exists", { for (conn in conns) { # nolint: brace_linter

File: tests/testthat/test-get_table.R [hunk 2]
```diff
@@ -64,3 +64,9 @@ test_that("get_table() works", { for (conn in conns) { # nolint: brace_linter
   t <- id("tset.mtcars", conn)
   expect_error(get_table(conn, t), regexp = "Table tset.mtcars is not found!")
 }})
+
+test_that("get_table returns list of tables if no table is requested", { for (conn in conns) {
```
Comment (RasmusSkytte) [inline tests/testthat/test-get_table.R:68]: I think this will also give a linting error. Check and add # nolint: brace_linter if needed

**Review by RasmusSkytte — CHANGES_REQUESTED** [review]
Please check that devtools::lint() finds nothing

**Review by RasmusSkytte — APPROVED** [review]
